lte/gateway/c/core/oai/tasks/mme_app/mme_app_context.cpp

```diff
@@ -766,11 +771,11 @@ void mme_remove_ue_context(mme_ue_context_t* const mme_ue_context_p,
     char guti_str[GUTI_STRING_LEN] = {0};
     convert_guti_to_string(&ue_context_p->emm_context._guti, &guti_str);
 
-    if (mme_ue_context_p->guti_ue_context_map.remove(guti_str) !=
+    if (mme_ue_context_p->mme_app_guti2mme_ue_id_map.remove(guti_str) !=
         magma::PROTO_MAP_OK) {
       OAILOG_ERROR_UE(
           LOG_MME_APP, ue_context_p->emm_context._imsi64,
-          "Failed to remove guti_ue_context_map for "
+          "Failed to remove mme_app_guti2mme_ue_id_map for "
```

Comment:
Rephrase as "Failed to remove guti from mme_app_guti2mme_ue_id_map" with other details